tests/test_machine.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import os
+
+import pytest
+
+import mkosi.machine as machine
+
+
+@pytest.mark.parametrize("verb", ["qemu", "boot"])
+@pytest.mark.skipif(os.getuid() != 0, reason="Must be invoked as root.")
+@pytest.mark.integration
+def test_run(verb: str) -> None:
+    boot = ["-d", "fedora", "-f", "--autologin"]
+
+    if verb == "qemu":
+        boot += ["--bootable", "--qemu-headless", "--hostonly-initrd", "--network-veth", "--ssh", verb]
+    if verb == ("boot"):
```

Comment:
All these string comparisons against the verb are getting rather brittle. A good refactoring would be to introduce an enum to represent the verb, similar to how we use an enum to represent the current distribution.